exovetter/utils.py

```diff
@@ -0,0 +1,13 @@
+import typing
+
+from scipy.io import loadmat
+from astropy.utils import data
+
+ENCODING = 'utf-8'
+
+
+def load_mat(url: str) -> typing.Dict[str, typing.Any]:
```

Comment:
* This needs a docstring to document the API.
* There is no need to check the URL, astropy does it for you.
* Also from what I understood from @fergalm , this function should be able to load a given local file, not just the URL. If a local file is given, it should be smart enough to bypass the download and caching.
* Would be nicer if it loads the `.mat` and then parse it to something we can use, but maybe that can be a separate PR after we have the requirements from @mustaric .

Reply:
* I'll upgrade the method to check the file-system and then pass the `url_or_filepath` to `astropy`
* I agree having the function parse the `mat` file into something we can use. We could also create a second function to parse the `.mat` file into something usable. It would also make it easier to write tests for